routes/nearbyInterestStack.js

```diff
@@ -15,7 +18,25 @@ const screens = {
         navigationOptions: {
             headerShown: false,
         }
-    }
+    },
```

Comment:
We would need @Firassawan  to check with Sonar if this doesn't introduce duplication. If it does, I suggest creating a javascript file called on which you put all the screens that are shared between drawerStacks and export it. And in the proper stacks, you just call that component instead of having the following screens written twice.

Reply:
Not a bad idea to avoid duplications. I'll check it out!

Reply:
![Image from iOS (5)](https://user-images.githubusercontent.com/46238950/79458766-49649b80-7fc0-11ea-95a6-78f1fecc38d9.png)

![image](https://user-images.githubusercontent.com/46238950/79459837-e96ef480-7fc1-11ea-87c7-38979e5ed4e5.png)

The suggestion to create a JavaScript file and export it **to avoid the duplication** of code for the shared screens between stacks of the draweStack **will not** be possible to implement .

After multiple attempts to concatenate the two constants and add render them in the createStackNavigator, the system does not accept it as shown in the first screenshot.

The code has be written **as** originally written in the nearbyInterestStack and scheduleStack
(with code duplication) 😢